idd/Energy+.idd.in

```diff
@@ -12801,11 +12809,14 @@ WindowShadingControl,
        \note Required if Type of Slat Angle Control for Blinds = ScheduledSlatAngle
        \note Schedule values should be degrees (0 minimum, 180 maximum)
   N3 , \field Setpoint 2
-       \units W/m2 or deg C
-       \note W/m2 for solar-based controls, deg C for temperature-based controls.
+       \units W/m2, deg C, or cd/m2
```

Comment:
IDF Editor does not recognize this units string and it will throw a warning every time it is opened for any idf file. @JasonGlazer Do we want to update IDF Editor at this point for a set of units that won't get converted? Or just remove the units field here?

Reply:
@mjwitte it is very simple to add units to the IDF Editor so I think we should try to do that.